### PR TITLE
CR-1034 - config.yml

### PR DESCRIPTION
**Pull request by egflores-conductor**
Correctly writes config to `%APPDATA%\Conductor Technologies\Conductor` Notes in [Jira](https://conductorio.atlassian.net/browse/CR-1034)

### CODE REVIEW

File: conductor/lib/common.py [hunk 1]
```diff
@@ -509,6 +509,8 @@ def get_config_file_paths(self, config_file=None):
             possible_paths = [x for x in os.environ['CONDUCTOR_CONFIG'].split(path_separator) if len(x) > 0]
```
Comment (lawschlosser) [inline conductor/lib/common.py]: not part of this commit, but see `os.path.pathsep` for cross platform path separator goodness

File: conductor/lib/common.py [hunk 2]
```diff
@@ -509,6 +509,8 @@ def get_config_file_paths(self, config_file=None):
...
             if len(possible_paths) > 0:
                 return possible_paths
+        if platform.system() == 'Windows':
+            return [os.path.join(os.environ['APPDATA'], 'Conductor Technologies', 'Conductor', 'config.yml')]
```
Comment (lawschlosser) [inline conductor/lib/common.py]: The client's preferences directory should really live as either a constant or retrieved by a getter function.  Not sure if the installer is responsible/knowledgeable of creating a user prefs directory for Conductor, but obviously the python code should by in sync with it (and ideally tightly bound to it, i.e. DRY).

File: conductor/lib/common.py (hunk 1 above)
Comment (flebel) [reply conductor/lib/common.py]: Tracking as https://conductorio.atlassian.net/browse/CR-1051

File: conductor/lib/common.py (hunk 2 above)
Comment (flebel) [reply conductor/lib/common.py]: @lawschlosser do you feel strongly enough about this to create a Jira ticket?

Comment (egflores-conductor) [reply conductor/lib/common.py]: this is going be addressed by CR-462

Comment (lawschlosser) [reply conductor/lib/common.py]: I hate writing jira tickets.... especially for other people.   yes, I do feel strongly about it ;)
